AppCenterDistribute/AppCenterDistribute/MSDistribute.m

```diff
@@ -717,46 +718,45 @@ - (BOOL)isNewerVersion:(MSReleaseDetails *)details {
 }
 
 - (void)storeDownloadedReleaseDetails:(nullable MSReleaseDetails *)details {
-  if (details == nil || details.id == nil || details.packageHashes == nil || [details.packageHashes count] == 0) {
-    MSLogDebug([MSDistribute logTag], @"Release details are missing or broken, will not store release hash and id for reporting.");
+  if (details == nil) {
+    MSLogDebug([MSDistribute logTag], @"Downloaded release details are missing or broken, won't store.");
     return;
   }
-
+  NSString *groupId = details.distributionGroupId;
   NSNumber *releaseId = details.id;
 
   /*
    * IPA can contain several hashes, each for different architecture and we can't predict which will be installed,
    * so save all hashes as comma separated string.
    */
-  NSString *releaseHashes = [details.packageHashes count] > 1
+  NSString *releaseHashes = [details.packageHashes count] > 0
           ? [details.packageHashes componentsJoinedByString:@","]
-          : details.packageHashes[0];
+          : nil;
+  MSLogDebug([MSDistribute logTag], @"Store downloaded release details, group id: %@, release hashes: %@, release id: %@.",
```

Comment:
Can you move this log back to the original place? This log will not tell us that all values are properly stored to `NSUserDefaults`.

Reply:
You are right, fixed. Thank you.